From 6213eb96adc9674586957805b92ffedbbf516953 Mon Sep 17 00:00:00 2001
From: Pino Toscano <toscano.pino@tiscali.it>
Date: Fri, 17 Apr 2020 13:25:12 +0200
Subject: [PATCH] Include <stdexcept> for std::runtime_error

It was implicitly pulled before, and in GCC 10 it will not be anymore.
---
 src/game/audio/openaloggdata.cpp | 1 +
 src/game/audio/openalsource.cpp  | 1 +
 src/game/audio/openalwavdata.cpp | 1 +
 3 files changed, 3 insertions(+)

diff --git a/src/game/audio/openaloggdata.cpp b/src/game/audio/openaloggdata.cpp
index 56634c2..4a2ab88 100644
--- a/src/game/audio/openaloggdata.cpp
+++ b/src/game/audio/openaloggdata.cpp
@@ -19,6 +19,7 @@
 
 #include <cstdio>
 #include <vector>
+#include <stdexcept>
 
 #include <vorbis/vorbisfile.h>
 
diff --git a/src/game/audio/openalsource.cpp b/src/game/audio/openalsource.cpp
index 9d2bb7f..37a4272 100644
--- a/src/game/audio/openalsource.cpp
+++ b/src/game/audio/openalsource.cpp
@@ -19,6 +19,7 @@
 #include <AL/alc.h>
 
 #include <memory>
+#include <stdexcept>
 
 static bool checkError()
 {
diff --git a/src/game/audio/openalwavdata.cpp b/src/game/audio/openalwavdata.cpp
index 3a9fc20..015a11d 100644
--- a/src/game/audio/openalwavdata.cpp
+++ b/src/game/audio/openalwavdata.cpp
@@ -25,6 +25,7 @@
 #include <AL/alc.h>
 #include <cassert>
 #include <cstdio>
+#include <stdexcept>
 
 static bool checkError()
 {
-- 
2.26.2

